### ISSUE TITLE
Responsive Layouts - Like Devices (sm, md, lg, xl) & Viewport dimensions

### ISSUE THREAD

**Issue by anburocky3** [microsoft/vscode-livepreview#116]
Wouldn't be great if we have responsive toggle buttons at the bottom so it would be cool for the UI developer to check the design on all screens. (`sm`, `md`, `lg`, `xl`, `2xl`)

Even, if we can see the window dimensions, we can understand the correct viewport of that webpage.

From the docs, we can  see, 
>  This allows support for features such as setting breakpoints and inspecting elements.

But i don't know how to do that. Displaying the viewport information on the bottom would be a good idea, I think.

**Comment by andreamah** [comment]
To run the debugger, try running `Live Preview: Show Debug Preview`. Here, you can access the Edge Devtools if you have [the extension](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) installed. You can open devtools via this button:
![image](https://user-images.githubusercontent.com/31675041/127202886-4df11391-65f8-4ec0-bd87-f1834940c237.png)

For the screen sizes, I'll add that to the feature backlog 👍

**Comment by anburocky3** [comment]
Would not be cool if we can display the current viewport dimensions like 

![image](https://user-images.githubusercontent.com/15264938/127206581-48305f33-012a-4d6c-ae95-4379ec6a378a.png)

**Comment by andreamah** [comment]
Sorry, I'm not too sure what you mean? Are you saying that the dimensions of the embedded preview are not ideal and it would be better to allow users to customize it according to the sizes you mentioned above?

**Comment by anburocky3** [comment]
No, while displaying the live preview on the right side, there is no indication of what is the current viewport of the live preview.

So what I'm suggesting is that it would be cool, if we have something like this.

> Page Title - (1080px X 720px) 

So just like chrome devtools, 
![image](https://user-images.githubusercontent.com/15264938/127218225-93491daa-864e-4dd9-a974-455bf170ebff.png) if we can display it after the page title, it would be useful for UI designers right..

**Comment by anburocky3** [comment]
I'm asking like this: or having it some kind of absolute will also be cool.
![image](https://user-images.githubusercontent.com/15264938/127227473-bb4356db-1cd8-4e11-b01d-c08a3e8ea592.png)

**Comment by anburocky3** [comment]
Or even, it would be cool, if we can implement this kind of behavior:

Check this: https://play.tailwindcss.com/

Whenever we resize it will display the viewports info for a sec.

**Comment by anburocky3** [comment]
I have created a quick prototype: 
![image](https://user-images.githubusercontent.com/15264938/127235225-19ca8632-1ec6-4b69-bd15-962bbfe25f21.png)

1. If they want to check responsiveness, they can click on the icon (next to input) this will act like a toggle.
2. This will display the responsive buttons based on the toggle.
